src/index.ts

```diff
@@ -17,9 +17,7 @@ export const templateDefaults = {
 };
 
 const renderer = (templates: Templates, name: FormatterName): FormatFn => {
-  const completeTemplates = { ...templateDefaults, templates };
-  const render = compile(completeTemplates[name], helpers);
-
```

Comment:
This was broken when extracting and changing the original implementation, see:
https://github.com/bitcrowd/tickety-tick/pull/287/files#diff-80bd531a236a6e714847cd8159ce7d165ab7c9f89888418cb8db2dbb50ee8791L13-L16

Reply:
1. Object spreading won't cut it here, cfg.:
    ```ts
    const a = { x: 0, y: 1 }
    const b = { x: undefined, y: 2 }
    { ...a, ...b } // => { x: undefined, y: 2 }
    ```
2. The second `templates` object was not spread, but a new, nested `"template"` property was added to the object, compare:
    ```ts
    { ...a, ...b } // => { x: undefined, y: 2 } as befoer
    { ...a, b } // => { x: 0, y: 1, b: { x: undefined, y: 2 } }
    ```